src/EnergyPlus/TarcogShading.cc

```diff
@@ -706,7 +706,7 @@ namespace TarcogShading {
         TGapOld1 = 0.0;
         TGapOld2 = 0.0;
         tilt = Constant::Pi / 180 * (angle - 90);
-        T0 = 0.0 + Constant::KelvinConv;
+        T0 = 0.0 + Constant::Kelvin;
```

Comment:
Huh? maybe just T0=Constant:Kelvin

Reply:
I'm pretty sure that the use of the conversion factor predates a lot of what is possible now with `constexpr` and friends. Maybe we could just define a function to do the temperature conversion at compile time for stuff like this now? And ditch the constant.

Reply:
This is all global search-replace stuff. I don't have opinions on these individual lines.